eclipsePlugin/build.gradle

```diff
@@ -34,6 +36,12 @@ def eclipsePluginId = 'com.github.spotbugs.plugin.eclipse'
 def localProps = new Properties()
 localProps.load(new FileInputStream("$projectDir/local.properties"))
 
+ext {
+  eclipseRootDir = localProps.getProperty('eclipseRoot.dir')
+  eclipseExecutable = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/MacOS/eclipse" : "/eclipse")
+  eclipsePluginsDir = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/Eclipse/plugins" : "/plugins")
+}
```

Comment:
I think we schould throw away 3.6, may be even 3.8 support. There is really no reason  to support ancient Eclipse releases if this makes the code more complex.